dora/core/common/src/main/java/alluxio/grpc/GrpcSerializationUtils.java

```diff
@@ -79,7 +79,7 @@ public class GrpcSerializationUtils {
       sBufferList = getPrivateField(BUFFER_CHAIN_OUTPUT_STREAM_CLASS_NAME, BUFFER_LIST_FIELD_NAME);
       sCurrent = getPrivateField(BUFFER_CHAIN_OUTPUT_STREAM_CLASS_NAME, CURRENT_FIELD_NAME);
       sCompositeBuffers =
-          getPrivateField(CompositeReadableBuffer.class.getName(), BUFFERS_FIELD_NAME);
+          getPrivateField(CompositeReadableBuffer.class.getName(), READABLE_BUFFERS_FIELD_NAME);
```

Comment:
https://github.com/grpc/grpc-java/pull/8102/files 
see this pr for the renaming change